src/qt/optionsmodel.cpp

```diff
@@ -110,6 +110,11 @@ void OptionsModel::Init(bool resetSettings)
         settings.setValue("bSpendZeroConfChange", true);
     if (!gArgs.SoftSetBoolArg("-spendzeroconfchange", settings.value("bSpendZeroConfChange").toBool()))
         addOverriddenOption("-spendzeroconfchange");
+
+    if (!settings.contains("strAddressType"))
+        settings.setValue("strAddressType", QString::fromStdString(FormatOutputType(OUTPUT_TYPE_DEFAULT)));
```

Comment:
We do that also at different places... but this makes changing default in the future really hard. If we the default for `-addresstype`, QT remains at the old state even if the user hasn't changed the value there...

Reply:
We can fix this in a follow up PR (that also fixes the other issues with defaults).